eventmesh-openconnect/eventmesh-openconnect-java/src/main/java/org/apache/eventmesh/openconnect/SourceWorker.java

```diff
@@ -330,15 +330,15 @@ public boolean commitOffsets() {
             log.info("{} Committing offsets for {} acknowledged messages", this, committableOffsets.numCommittableMessages());
             if (committableOffsets.hasPending()) {
                 log.debug("{} There are currently {} pending messages spread across {} source partitions whose offsets will not be committed. "
-                        + "The source partition with the most pending messages is {}, with {} pending messages",
+                    + "The source partition with the most pending messages is {}, with {} pending messages",
```

Comment:
Our community has always preferred using 1TAB instead of 2TAB, and my IDEA's default newline indentation is also 1TAB. This time, spotless mainly formatted the code for lark and wecom connector.